js/main.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var HOTELSCOUNT = 8;
```

Comment:
Слова в константах лучше разделять нижним подчеркиванием `_`
```suggestion
var HOTELS_COUNT = 8;
```